src/SFA.DAS.EmployerAccounts.Web/Controllers/SearchPensionRegulatorController.cs

```diff
@@ -208,22 +208,14 @@ private void ValidateAndFormatSearchPensionRegulatorByAornViewModel(SearchPensio
         viewModel.Aorn = viewModel.Aorn?.ToUpper().Trim();
         viewModel.PayeRef = viewModel.PayeRef?.ToUpper().Trim();
 
-        if (string.IsNullOrWhiteSpace(viewModel.Aorn))
+        if (string.IsNullOrWhiteSpace(viewModel.Aorn) || !_aornRegex.IsMatch(viewModel.Aorn.ToUpper().Trim()))
         {
-            errors.Add(nameof(viewModel.Aorn), "Enter your reference number to continue");
-        }
-        else if (!_aornRegex.IsMatch(viewModel.Aorn.ToUpper().Trim()))
-        {
-            errors.Add(nameof(viewModel.Aorn), "Enter an accounts office reference number in the correct format");
+            errors.Add(nameof(viewModel.Aorn), "You need to enter your Accounts Office reference in the correct format for example 123PX00123456 or 123PX0012345X");
         }
 
-        if (string.IsNullOrWhiteSpace(viewModel.PayeRef))
-        {
-            errors.Add(nameof(viewModel.PayeRef), "Enter your PAYE scheme to continue");
-        }
-        else if (!_payeRegex.IsMatch(viewModel.PayeRef))
+        if (string.IsNullOrWhiteSpace(viewModel.PayeRef) || !_payeRegex.IsMatch(viewModel.PayeRef))
         {
-            errors.Add(nameof(viewModel.PayeRef), "Enter a PAYE scheme number in the correct format");
+            errors.Add(nameof(viewModel.PayeRef), "You need to enter your PAYE in the correct format For example 123/AB456");

```

Comment:
This looks like it matches the content in the ticket, but the capitalised `F` mid-sentence feels wrong.